bot/cogs/guess_number.py

```diff
@@ -0,0 +1,100 @@
+import discord
+import math, random
+from discord.ext.commands import Bot, Cog, command
+
+
+class guess_game(Cog): #(Cog): # Include Cog if it's a part of Cog file in Friendo Bot. Change class name if needed.
```

Comment:
remove comments like this before pushing, this could be a comment on the PR or issue if clarification was needed.